docs/configuration-reference/components/rook-ceph.md

```diff
@@ -11,7 +11,7 @@
 
 ## Introduction
 
-[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component installs a Ceph
+[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component applys a Ceph
```

Comment:
```suggestion
[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component applies a Ceph
```

Reply:
This is actually not correct. I think `installs` is correct in this case.

Reply:
heh, search and replace :see_no_evil: